### ISSUE TITLE
Unable to get metrics from kubelet for static pods with cri-o

### ISSUE THREAD

**Issue by matthias50** [kubernetes/kubernetes#80181]
**What happened**:

usageNanoCores and usageCoreNanoSeconds are always reported as 0 at the container level. There is no per container memory being reported. Overall pod usageNanoCores is also always 0

**What you expected to happen**:
Static pod CPU and memory metrics are available from kubelet as below for a node using Docker:

```
curl localhost:10255/stats/summary
...
{
  "podRef": {
	"name": "kube-proxy-ip-<redacted>.us-west-2.compute.internal",
	"namespace": "kube-system",
	"uid": "31dcb33320163f7de65355bd033bea51"
  },
  "startTime": "2019-07-11T16:11:33Z",
  "containers": [
	{
	  "name": "kube-proxy",
	  "startTime": "2019-07-11T16:11:36Z",
	  "cpu": {
		"time": "2019-07-15T20:46:42Z",
		"usageNanoCores": 3848339,
		"usageCoreNanoSeconds": 1059148038921
	  },
	  "memory": {
		"time": "2019-07-15T20:46:42Z",
		"usageBytes": 19574784,
		"workingSetBytes": 19300352,
		"rssBytes": 14868480,
		"pageFaults": 7458,
		"majorPageFaults": 0
	  },
	  "rootfs": {
		"time": "2019-07-15T20:46:42Z",
		"availableBytes": 255627218944,
		"capacityBytes": 257568018432,
		"usedBytes": 0,
		"inodesFree": 125764336,
		"inodes": 125827072,
		"inodesUsed": 11
	  },
	  "logs": {
		"time": "2019-07-15T20:46:42Z",
		"availableBytes": 80046202880,
		"capacityBytes": 84417077248,
		"usedBytes": 40960,
		"inodesFree": 5242750,
		"inodes": 5242880,
		"inodesUsed": 130
	  },
	  "userDefinedMetrics": null
	}
  ],
  "cpu": {
	"time": "2019-07-15T20:46:32Z",
	"usageNanoCores": 1821610,
	"usageCoreNanoSeconds": 1059104123535
  },
  "memory": {
	"time": "2019-07-15T20:46:32Z",
	"usageBytes": 20381696,
	"workingSetBytes": 20107264,
	"rssBytes": 14876672,
	"pageFaults": 0,
	"majorPageFaults": 0
  },
```
**How to reproduce it (as minimally and precisely as possible)**:

```
curl localhost:10255/stats/summary
...
{
  "podRef": {
	"name": "kube-proxy-ip-<redacted>.us-west-2.compute.internal",
	"namespace": "kube-system",
	"uid": "31dcb33320163f7de65355bd033bea51"
  },
  "startTime": "2019-07-15T20:12:49Z",
  "containers": [
	{
	  "name": "kube-proxy",
	  "startTime": "2019-07-15T20:12:49Z",
	  "cpu": {
		"time": "2019-07-15T20:48:14Z",
		"usageNanoCores": 0,
		"usageCoreNanoSeconds": 0
	  },
	  "rootfs": {
		"time": "2019-07-15T20:48:14Z",
		"availableBytes": 255673331712,
		"capacityBytes": 257568018432,
		"usedBytes": 0,
		"inodesFree": 125767223,
		"inodes": 125827072,
		"inodesUsed": 15
	  },
	  "logs": {
		"time": "2019-07-15T20:48:14Z",
		"availableBytes": 80045953024,
		"capacityBytes": 84417077248,
		"usedBytes": 8192,
		"inodesFree": 5242605,
		"inodes": 5242880,
		"inodesUsed": 275
	  },
	  "userDefinedMetrics": null
	}
  ],
  "cpu": {
	"time": "2019-07-15T20:48:15Z",
	"usageNanoCores": 0,
	"usageCoreNanoSeconds": 57507302265
  },
  "memory": {
	"time": "2019-07-15T20:48:15Z",
	"usageBytes": 12767232,
	"workingSetBytes": 12476416,
	"rssBytes": 8359936,
	"pageFaults": 0,
	"majorPageFaults": 0
  },
```
**Anything else we need to know?**:

**Environment**:
- Kubernetes version (use `kubectl version`): 
Client Version: version.Info{Major:"1", Minor:"13", GitVersion:"v1.13.7", GitCommit:"4683545293d792934a7a7e12f2cc47d20b2dd01b", GitTreeState:"clean", BuildDate:"2019-06-06T01:46:52Z", GoVersion:"go1.11.5", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"13", GitVersion:"v1.13.7", GitCommit:"4683545293d792934a7a7e12f2cc47d20b2dd01b", GitTreeState:"clean", BuildDate:"2019-06-06T01:39:30Z", GoVersion:"go1.11.5", Compiler:"gc", Platform:"linux/amd64"}
- Cloud provider or hardware configuration: AWS
- OS (e.g: `cat /etc/os-release`): 
NAME="CentOS Linux"
VERSION="7 (Core)"
ID="centos"
ID_LIKE="rhel fedora"
VERSION_ID="7"
PRETTY_NAME="CentOS Linux 7 (Core)"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:centos:centos:7"
HOME_URL="https://www.centos.org/"
BUG_REPORT_URL="https://bugs.centos.org/"

CENTOS_MANTISBT_PROJECT="CentOS-7"
CENTOS_MANTISBT_PROJECT_VERSION="7"
REDHAT_SUPPORT_PRODUCT="centos"
REDHAT_SUPPORT_PRODUCT_VERSION="7"
- Kernel (e.g. `uname -a`): Linux ip-10-121-2-62.us-west-2.compute.internal 5.1.16-1.el7.elrepo.x86_64 #1 SMP Wed Jul 3 10:53:38 EDT 2019 x86_64 x86_64 x86_64 GNU/Linux
- Install tools: Custom stack using terraform/puppet
- Network plugin and version (if this is a network-related bug):
- Others:

**Comment by matthias50** [comment]
Note that this appears to be different from https://github.com/kubernetes/kubernetes/pull/73659 since this bug was fixed in 1.13.7 and I am running 1.13.7. I can successfully query the same stats for other, non-static pods on the same node where I could not get stats for the static kube-proxy pod.

**Comment by matthias50** [comment]
Not seeing this in 1.14.7.
